src/controllers/controllermanager.h

```diff
@@ -30,6 +30,8 @@ class ControllerManager : public QObject {
     ControllerManager(UserSettingsPointer pConfig);
     virtual ~ControllerManager();
 
+    static const mixxx::Duration kPollInterval;
```

Comment:
Why does this constant need to be defined as a public static member of the class? It is only used in the .cpp file.

Reply:
I was going to use it in HidController but ended up not. It should be used in ControllerEngine too for jog wheel scratching, but that's beyond the scope of this PR.